types/mempool/sender_nonce.go

```diff
@@ -183,7 +196,7 @@ func (snm *senderNonceMempool) Remove(tx sdk.Tx) error {
 	}
 
 	sig := sigs[0]
-	sender := sig.PubKey.Address().String()
+	sender := sdk.AccAddress(sig.PubKey.Address()).String()
```

Comment:
we shouldnt use bech32, this will move from being a global to local to auth. At least the mempool should not sue the global as a bare minimuim

Reply:
Bech32 isn't going anywhere. When/if we figure out a tenable replacement for the global, that'll simple be passed into the mempool constructor. Bech32 is the correct approach for sender indexes.